src/main.cpp

```diff
@@ -4,13 +4,22 @@
 #include <QApplication>
 #include <QTranslator>
 
+#ifndef INSTALL_PREFIX
+#define INSTALL_PREFIX "/usr/local"
+#endif
+
+#define xstr(s) str(s)
+#define str(s) #s
+
 // -----------------------------------------------------------------------
 int main(int argc, char *argv[])
 {
 	QApplication a(argc, argv);
 
 	QTranslator trans;
-	trans.load(QLocale(), "krokonf", "_", ".");
+	if (!trans.load(QLocale(), "krokonf", "_", xstr (INSTALL_PREFIX)"/share/krokonf/translations/")) {
+		trans.load(QLocale(), "krokonf", "_", ".");
```

Comment:
Tego również można się pozbyć. Takie ładowanie tłumaczeń wynikało z tego, że są w fazie "jak to się robi w QT". :-)

Reply:
OK